backend/api/quivr_api/modules/chat/controller/chat_routes.py

```diff
@@ -271,6 +282,8 @@ async def create_stream_question_handler(
     for model in models:
         if brain_id == generate_uuid_from_string(model.name):
```

Comment:
I don't get this function. This seems complicated for getting model ?

Reply:
This is @StanGirard quick-and-dirty approach to get the models in chat-with-llm mode, we will certainly find a better way in the future